### PR TITLE
Add base work environments

### PR DESCRIPTION
**Pull request by vademyanchik**
Add vue-cli webpack-simple template.
Commands - 
`npm install -g vue-cli`
`vue init webpack-simple alis-web-vue`

### CODE REVIEW

**Comment by irnc** [comment]
@vademyanchik Have you used some CLI tool to create part that code? Could you please share which command and exactly you have used?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
 # alis-web-vue
 Experimental UI for ALIS WEB data created using Vue.js
+
+# Goal
```
Comment (irnc) [inline README.md]: I would propose to label this heading from a user point of view, so instead of _Goal_, we say _What it does?_

File: README.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
 # alis-web-vue
 Experimental UI for ALIS WEB data created using Vue.js
+
+# Goal 
+
+Create client UI for library. Whose allows see latest added books.
```
Comment (irnc) [inline README.md]: Better spelling telling the same would be _`alis-web-vue` is a web app which shown latest additions to a public library._

**Comment by vademyanchik** [comment]
@irnc 
`npm install -g vue-cli`
`vue init webpack-simple alis-web-vue`

https://github.com/vuejs/vue-cli

**Pull request merged**